core/services/keystore/keys/ocr2key/offchain_keyring.go

```diff
@@ -47,6 +50,25 @@ func newOffchainKeyring(encryptionMaterial, signingMaterial io.Reader) (*Offchai
 	return ok, nil
 }
 
+// Decrypt decrypts a message that was encrypted using the public key
+func (ok *OffchainKeyring) Decrypt(ciphertext []byte) (plaintext []byte, err error) {
```

Comment:
I would suggest using a more precise name here, e.g. `NaclBoxOpenAnonymous`. After all there are any decryption schemes that could be used here.